README.md

```diff
@@ -55,4 +56,9 @@ EventMachine::HttpRequest.new("#{@config.base_url}/gif", options).post :body =>
   :meme_bottom => meme_bottom,
   :seekrit => @config.password # Minimal security lololol
 }
-```
\ No newline at end of file
+```
+
+Alternatively, here's a simple curl:
+```bash
+curl --data 'url=http://www.example.com/somegif.gif&who=thatAmazingPerson&meme_top=herp&meme_bottom=derp&seekrit=yourSuperSecretPasswordFromAppRb' 'http://yourGifMachineUrl/gif'
```

Comment:
hmm...maybe I should use bash environment variables and plug them into the `--data` here instead so it's more easy to plug & play like the ruby snippet above?